src/lib/react/useRoomConnection.ts

```diff
@@ -379,7 +379,17 @@ export type RoomConnectionRef = {
     _ref: RoomConnection;
 };
 
-export function useRoomConnection(roomUrl: string, roomConnectionOptions: UseRoomConnectionOptions): RoomConnectionRef {
+const defaultRoomConnectionOptions: UseRoomConnectionOptions = {
+    localMediaConstraints: {
+        audio: true,
+        video: true,
+    },
+};
+
+export function useRoomConnection(
+    roomUrl: string,
+    roomConnectionOptions = defaultRoomConnectionOptions
```

Comment:
I wonder if we should move the default options into the body so people could include some options without localMediaConstraints?

something like 
```typescript
export function useRoomConnection(
    roomUrl: string,
    roomConnectionOptions: UseRoomConnectionOptions;
) {
    const [roomConnection] = useState<RoomConnection>(
        () =>
            new RoomConnection(roomUrl, {
                ...defaultRoomConnectionOptions,
                ...roomConnectionOptions,
                localMedia: roomConnectionOptions?.localMedia?._ref || undefined,
            })
      );
}

Though I can see how that could be confusing for the end user

Reply:
Yeah, both solutions have pros and cons. 🤔 Personally, I find the currently committed one to be more straightforward to use. I mean, if I provide options, only those options will be used. However, if you feel that your suggestions fit better, I'm happy to change it. I can see its benefits too. 😅

Reply:
nah lets not bother, thinking about it more I agree with you